docs-chef-io/content/inspec/resources/user.md

```diff
@@ -43,108 +44,126 @@ A `user` resource block declares a user name, and then one (or more) matchers:
       its('badpasswordattempts') { should eq 0 }
       its('lastlogin') { should eq nil } // Only valid on Windows OS
     end
+```
 
-where
-
-- `('root')` is the user to be tested
-- `it { should exist }` tests if the user exists
-- `gid`, `group`, `groups`, `home`, `maxdays`, `mindays`, `shell`, `uid`, `warndays`´, `passwordage`, `maxbadpasswords`, `badpasswordattempts` and `lastlogin` are valid matchers for this resource
+> Where
+> - `('root')` is the user to be tested.
+> - `it { should exist }` tests if the user exists.
+> - `gid`, `group`, `groups`, `home`, `maxdays`, `mindays`, `shell`, `uid`, `warndays`´, `passwordage`, `maxbadpasswords`, `badpasswordattempts`, and `lastlogin` are valid matchers for this resource.
 
 ## Properties
 
 ### gid
 
-The `gid` property tests the group identifier:
+The `gid` property tests the group identifier.
 
     its('gid') { should eq 1234 }
 
-where `1234` represents the user identifier.
-The `gid` option is only available on Linux and will return `nil` for Windows os.
+> Where `1234` represents the user identifier.
+>
+> The `gid` option is only available on Linux and returns `nil` for Windows OS (Operating System).
```

Comment:
The indentation should not extend to this sentence.

Reply:
sure